internal/cri/server/container_create.go

```diff
@@ -87,6 +87,9 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	if metadata == nil {
 		return nil, errors.New("container config must include metadata")
 	}
+	if sandboxConfig.GetMetadata() == nil {
+		return nil, errors.New("pod sandbox config must include metadata")
+	}
 	containerName := metadata.Name
 	name := makeContainerName(metadata, sandboxConfig.GetMetadata())
```

Comment:
nit: avoid repeated calls.

```suggestion
    sandboxMetadata := sandboxConfig.GetMetadata()
	if sandboxMetadata == nil {
		return nil, errors.New("pod sandbox config must include metadata")
	}
	containerName := metadata.Name
	name := makeContainerName(metadata, sandboxMetadata)
```